apps/landing/src/index.css

```
@@ -18,15 +22,17 @@
   --font-sans:
     "Manrope Variable", "Manrope", ui-sans-serif, system-ui, sans-serif;
   --font-display:
-    "Manrope Variable", "Manrope", ui-sans-serif, system-ui, sans-serif;
+    "Unbounded", "Manrope Variable", ui-sans-serif, system-ui, sans-serif;
+  --font-serif: "Piazzolla", Georgia, "Times New Roman", serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)